evil-easymotion.el

```diff
@@ -278,7 +278,7 @@
     :scope ,scope
     :all-windows ,all-windows
     :initial-point ,initial-point
-    :push-jump ,push-jump
+    ,@(if push-jump-supplied? (list :push-jump push-jump) nil)
```

Comment:
we can use `when` here since `when` returns nil when the condition is falsy

```suggestion
    ,@(when push-jump-supplied? (list :push-jump push-jump))
```

Reply:
You're right, thanks for your suggestion.